mitiq/pec/pec.py

```diff
@@ -109,8 +109,12 @@ def execute_with_pec(
             f" but precision is {precision}."
         )
 
+    converted_circuit, _ = convert_to_mitiq(circuit)
```

Comment:
Are we sure the speedup is due to the "batching" and not just this line?

Reply:
IIUC, this saves us `num_samples` conversions from QPROGRAM -> Cirq by only doing it once. (We still have `num_samples` conversions from Cirq -> QPROGRAM in `sample_circuit` which we cannot avoid.) I'm ok with changing the behavior of `sample_sequence` and `sample_circuit`, but I'd rather not if all the speedup is coming from the initial Cirq conversion.

Reply:
Actually I just moved this line a few steps earlier, in order to use the converted circuit also to evaluate the one-norm.
However that conversion line was already in the code (it was added by the Standford students) and the actual sampling was already happening with `converted_circuit` instead of `circuit`. 

So I am quite sure that the speedup is due to the new sampling functions.

Reply:
Just documenting: We discussed this @ Mitiq meeting and I understand what's going on now - thanks!